kong/client.go

```diff
@@ -357,6 +357,22 @@ func (c *Client) Status(ctx context.Context) (*Status, error) {
 	return &s, nil
 }
 
+// Config returns the config of a Kong node. The return value is an interface,
+// and needs to be parsed by the caller.
+func (c *Client) Config(ctx context.Context) (any, error) {
+	req, err := c.NewRequest("GET", "/config", nil, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	var config map[string]any
+	_, err = c.Do(ctx, req, &config)
+	if err != nil {
+		return nil, err
+	}
+	return (config)["config"], nil
```

Comment:
I'm wondering whether it wouldn't be better to return raw config `[]byte`s instead of an interface. Having `[]byte` returned, we can tell the caller they have to unmarshal the underlying json (or yaml?). Returning just an interface, we return a nested go map which is already unmarshaled and transforming this into proper config structs will be painful on the caller side.

Reply:
Yep, I agree. I returned `[]byte` so that it can be directly unmarshaled by the caller.